### PR TITLE
Clean up some extra 'dealii::'s.

### PR DESCRIPTION
**Pull request by drwells**

### CODE REVIEW

File: include/deal.II/dofs/dof_accessor.templates.h [hunk 1]
```diff
@@ -1980,10 +1980,6 @@ template <int spacedim, bool level_dof_access>
 inline unsigned int
 DoFAccessor<0, 1, spacedim, level_dof_access>::n_active_fe_indices() const
 {
-  Assert((std::is_same<DoFHandler<1, spacedim>,
-                       dealii::DoFHandler<1, spacedim>>::value == true),
-         ExcNotImplemented());
```
Comment (marcfehling) [inline include/deal.II/dofs/dof_accessor.templates.h:1985]: That is an interesting remnant of the `hp::DoFHandler` days :)

File: include/deal.II/dofs/dof_accessor.templates.h [hunk 2]
```diff
@@ -2140,7 +2125,7 @@ namespace internal
         const DoFCellAccessor<dim, spacedim, level_dof_access> &accessor)
       {
         if (accessor.dof_handler->hp_capability_enabled == false)
-          return 0; // ::DoFHandler only supports a single active FE with index
+          return 0; // DoFHandler only supports a single active FE with index
```
Comment (marcfehling) [inline include/deal.II/dofs/dof_accessor.templates.h:2128]: These comments will be removed in #14866.

**Review by marcfehling — APPROVED** [review]
Looks good!

**Comment by marcfehling** [comment]
Some doxygen warnings
```
[100%] Built target documentation
/home/runner/work/dealii/dealii/source/base/quadrature_lib.cc:2251: warning: no matching class member found for 
  template Quadrature< 3 > QSimplex< 2 >::compute_affine_transformation(const std::array< Point< 3 >, 2+1 > &vertices) const

/home/runner/work/dealii/dealii/source/base/quadrature_lib.cc:2263: warning: no matching class member found for 
  template Quadrature< 2 > QSimplex< 2 >::mapped_quadrature(const std::vector< std::array< Point< 2 >, 2+1 >> &simplices) const

/home/runner/work/dealii/dealii/source/base/quadrature_lib.cc:2267: warning: no matching class member found for 
  template Quadrature< 3 > QSimplex< 2 >::mapped_quadrature(const std::vector< std::array< Point< 3 >, 2+1 >> &simplices) const

warning: Inheritance graph for 'SmartPointer' not generated, too many nodes (138), threshold is 50. Consider increasing DOT_GRAPH_MAX_NODES.
warning: Inheritance graph for 'Subscriptor' not generated, too many nodes (195), threshold is 50. Consider increasing DOT_GRAPH_MAX_NODES.
```

File: include/deal.II/dofs/dof_accessor.templates.h (hunk 2 above)
Comment (drwells) [reply include/deal.II/dofs/dof_accessor.templates.h:2128]: I'll undo these edits - thanks for catching that.

**Review by kronbichler — APPROVED** [review]

File: include/deal.II/dofs/dof_accessor.templates.h (hunk 1 above)
Comment (drwells) [reply include/deal.II/dofs/dof_accessor.templates.h:1985]: It is a pretty funny assertion!

File: include/deal.II/multigrid/multigrid.h [hunk 3]
```diff
@@ -709,15 +709,14 @@ namespace internal
...
-      const std::vector<const dealii::DoFHandler<dim> *> &dof_handler_vector,
-      dealii::Multigrid<VectorType> &                     multigrid,
-      const TRANSFER &                                    transfer,
-      OtherVectorType &                                   dst,
-      const OtherVectorType &                             src,
-      const bool                          uses_dof_handler_vector,
-      const typename dealii::mg::Signals &signals,
-      int)
+    vmult(const std::vector<const DoFHandler<dim> *> &dof_handler_vector,
+          dealii::Multigrid<VectorType> &             multigrid,
```
Comment (tjhei) [inline include/deal.II/multigrid/multigrid.h:713]: There are more dealii mentions here...

File: include/deal.II/multigrid/multigrid.h [hunk 4]
```diff
@@ -741,15 +740,14 @@ namespace internal
...
-      const std::vector<const dealii::DoFHandler<dim> *> &dof_handler_vector,
-      dealii::Multigrid<VectorType> &                     multigrid,
-      const TRANSFER &                                    transfer,
-      OtherVectorType &                                   dst,
-      const OtherVectorType &                             src,
-      const bool                          uses_dof_handler_vector,
-      const typename dealii::mg::Signals &signals,
-      ...)
+    vmult(const std::vector<const DoFHandler<dim> *> &dof_handler_vector,
+          dealii::Multigrid<VectorType> &             multigrid,
```
Comment (tjhei) [inline include/deal.II/multigrid/multigrid.h:744]: And here

**Review by tjhei — APPROVED** [review]

**Pull request merged**